packages/cli/test/integration/shared/utils/testServer.ts

```diff
@@ -177,6 +177,7 @@ export const setupTestServer = ({
 				variables: { controller: variablesController, path: 'variables' },
 			};
 
+			config.set('publicApi.swaggerUi.disabled', true);
```

Comment:
[this was accidentally removed over 3 months ago. ](https://github.com/n8n-io/n8n/pull/6314/files#diff-889fb30882d6aae0146df3148ae6d9fde164e91d53fe8ec7abea02b6a039eb36L23)